aws/src/main/java/org/apache/iceberg/aws/glue/GlueCatalog.java

```diff
@@ -162,6 +164,13 @@ private FileIO initializeFileIO(Map<String, String> properties) {
     }
   }
 
+  @VisibleForTesting
+  void initialize(String name, String path, AwsProperties properties, GlueClient client,
+      LockManager lock, FileIO io, Map<String, String> catalogProps) {
```

Comment:
Nit / non-blocking: The indentation for this seems a little weird to me.

If all of the properties are on the next line, will it fit? I do see conflicting things at times so it's not a huge deal, but I believe the preferred way would be something more like:

```java
  @VisibleForTesting
  void initialize(String name, String path, AwsProperties properties, GlueClient client,
                          LockManager lock, FileIO io, Map<String, String> catalogProps) {
```

But again, I see different formats so without clarification, I'd either go with the above, a single line (the next line) if they _all_ fit, or just leave it unless somebody states otherwise. I'll follow up for more clarification, but this is absolutley non-blocking.

Reply:
Agreed. The starting indent of all argument lines should match. That could be starting after `initialize(` or using a continuation indent on the following line. But the important thing is that all lines with arguments indent arguments at the same place.